### PR TITLE
fix: query handling on blockchain-explorer hook

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-5207

### CODE REVIEW

**Review by dennyscode — COMMENTED** [review]
Let´s see if that fixes it from now, on!

File: packages/dapp/src/hooks/useBlockchainExplorerURL.ts [hunk 1]
```diff
@@ -3,14 +3,12 @@ import { useWallet } from '../providers';
...
+  const { isSuccess, getChainById } = useChains();
 
-  if (account?.address && account?.chainId) {
+  if (account.isActive && account.chainId) {
     const chain = getChainById(account.chainId);
-    if (chain?.metamask) {
+    if (isSuccess) {
       return `${chain.metamask.blockExplorerUrls[0]}address/${account.address}`;
-    } else {
-      console.error(`No blockchain explorer found for ${account.chainId}`);
```
Comment (Abhikumar98) [inline packages/dapp/src/hooks/useBlockchainExplorerURL.ts:13]: let's leave this error log here.
It's tends to be informative imo if something goes wrong.

The flow of execution was being interrupted due to `throw Error(...)` https://github.com/lifinance/jumper.exchange/blob/2c3148e060b233b01ff889cac4cfce75ebec9137/packages/dapp/src/hooks/useChains.ts#L30

Comment (dennyscode) [reply packages/dapp/src/hooks/useBlockchainExplorerURL.ts:13]: @Abhikumar98 Sounds legit, I added it back

File: packages/dapp/src/hooks/useBlockchainExplorerURL.ts [hunk 2]
```diff
@@ -3,11 +3,11 @@ import { useWallet } from '../providers';
 
 export const useBlockchainExplorerURL = () => {
   const { account } = useWallet();
-  const { getChainById } = useChains();
+  const { isSuccess, getChainById } = useChains();
```
Comment (Addminus) [inline packages/dapp/src/hooks/useBlockchainExplorerURL.ts:6]: can we make this bit more readable by renaming isSuccess in this file?
`const {isSuccess: chainsLoaded, getChainById} = useChains();`

**Review by Addminus — CHANGES_REQUESTED** [review]

**Review by Addminus — APPROVED** [review]

**Review by chybisov — APPROVED** [review]

**Pull request merged**